src/distance/SO2Angular.cpp

```diff
@@ -25,9 +25,9 @@ double SO2Angular::distance(
     const aikido::statespace::StateSpace::State* _state2) const
 {
   // Difference between angles
-  double diff = mStateSpace->getAngle(
+  double diff = mStateSpace->toAngle(
```

Comment:
Nit: Are we correcting the variable names in this PR or not? (`_state`)

Reply:
I was doing it only in the `SO2.cpp/hpp` since I was mostly modifying them for this PR. I think I'll leave that for another PR and just have SO2 stuff cleaned up in code style.